include/gatekeeper_gt.h

```diff
@@ -74,6 +74,8 @@ struct gt_instance {
 	 * received fragments of the packet.
 	 */
 	struct rte_ip_frag_tbl *frag_tbl;
+	struct acl_search *acl4;
```

Comment:
Add a blank line before the line above since it has nothing to do with the fragment table.